dist/index.js

```diff
@@ -0,0 +1,1955 @@
+module.exports =
```

Comment:
IIUC this is autogenerated. Is that correct?

Reply:
Yep.